Solutions/TransmitSecurity/Data/Solution_TransmitSecurity.json

```diff
@@ -2,7 +2,7 @@
 {
     "Name": "TransmitSecurity",
     "Author": "Transmit Security",
-    "Logo": "<img src=\"https://portal.transmitsecurity.io/assets/logo-c16ca289.svg\" width=\"75px\" height=\"75px\">",
```

Comment:
replace logo link with this link 

"<img src=\"https://github.com/Azure/Azure-Sentinel/raw/refs/heads/master/Logos/Transmit_Security_Logo.svg\" width=\"75px\" height=\"75px\">",

Reply:
Done